FOS.Users.Api/FOS.Users.Api/FOS.Users.Api/Controllers/UserManagementController.cs

```diff
@@ -0,0 +1,177 @@
+                    Message = lookup
+                });
+            }
+            catch (Exception ex)
+            {
+                return ErrorResponse(new Models.Responses.FOSMessageResponse
+                {
+                    StatusCode = System.Net.HttpStatusCode.BadRequest,
+                    Error = new FOSErrorResponse { Exception = ex }
+
+                });
+            }
+        }
+
+
+        /// <summary>
+        /// Get the UserDesignationLevel Lookup.
+        /// </summary>
+        [HttpPost]
+        [Route("GetUserdesignationlevel")]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(byte[]), StatusCodes.Status200OK, Web.ContentType.Json)]
+        [ProducesResponseType(typeof(FOSBaseResponse), StatusCodes.Status400BadRequest, Web.ContentType.Json)]
+        [ProducesResponseType(typeof(FOSBaseResponse), StatusCodes.Status500InternalServerError, Web.ContentType.Json)]
+
+        public async Task<IActionResult> GetUserdesignationlevelLookup()
+        {
+            try
+            {
+                var query = new GetUserdesignationlevelLookups.Query();
+                var lookup = await FOSMediator.Send(query);
+
+                return Ok(new FOSResponse
+                {
+                    Status = Status.Success,
+                    Message = lookup
+                });
+            }
+            catch (Exception ex)
+            {
+                return ErrorResponse(new Models.Responses.FOSMessageResponse
+                {
+                    StatusCode = System.Net.HttpStatusCode.BadRequest,
+                    Error = new FOSErrorResponse { Exception = ex }
+
+                });
+            }
+        }
+
+
+
+
+
+        /// <summary>
+        /// Creates a User Details ,Insert.
+        /// </summary>
+        /// <param name="prospectRequest">Prospect Request.</param>
+        /// <returns>A <see cref="Task{IActionResult}"/> representing the result of the asynchronous operation.</returns>
+        /// <response code="200">Returns the user's requests as a byte array.</response>
+        /// <response code="400">If the query is invalid or the message handler response status is not OK.</response>
+        /// <response code="401">Returns if the user is unauthorized.</response>
+        /// <response code="500">If an internal server error occurs.</response>
+        [HttpPost]
+        [Route("UserInsert")]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]       
+        [ProducesResponseType(typeof(byte[]), StatusCodes.Status200OK, Web.ContentType.Json)]
+        [ProducesResponseType(typeof(FOSBaseResponse), StatusCodes.Status400BadRequest, Web.ContentType.Json)]
+        [ProducesResponseType(typeof(FOSBaseResponse), StatusCodes.Status500InternalServerError, Web.ContentType.Json)]
+
+        public async Task<IActionResult> UserDetailsInsert(UserInsertDetailsModel newUserdetails)
```

Comment:
Make the Naming conventions of a endpoint to be a Verb. "InsertUserDetails", "GetUserDetails"etc.